apps/next/components/user/login-form.tsx

```diff
@@ -65,14 +90,11 @@ export const LoginForm: React.FC = () => {
         })
   }
 
-  const digest = getDigest(signIn.error) ?? getDigest(passwordlessSignIn.error)
-
   return (
     <LoginFormComponent
       signIn={handleSignIn}
       isPending={signIn.isPending || passwordlessSignIn.isPending}
-      isError={signIn.isError || passwordlessSignIn.isError}
-      errorMessage={`Log in was not successful, please try again; ref: ${digest}`}
+      {...getErrorProps(passwordlessSignIn.error, signIn.error, error ?? null)}
```

Comment:
To have clear what the component props are can we call the function to assign a variable and then pass in the variable down here?

const { isError, errorMessage } = getErrorProps( { error ,searchParamsError } )

isError={isError}
errorMessage={errorMessage}